src/templates/plugin.routing.js

```diff
@@ -86,7 +87,7 @@ function resolveRoute (route, locale) {
       if (isPrefixed) {
         localizedRoute.path = `/${locale}${localizedRoute.path}`
       }
-      localizedRoute.path = localizedRoute.path.replace(/\/+$/, '') + (nuxtOptions.trailingSlash ? '/' : '') || '/'
+      localizedRoute.path = nuxtOptions.trailingSlash ? withTrailingSlash(localizedRoute.path, true) : withoutTrailingSlash(localizedRoute.path, true)
```

Comment:
What about the case of `/` path and `trailingSlash = false`? Does this end up with empty string or `/` (as it should)?

Reply:
It's ending up as `"/"`